src/task_queue/tasks.py

```diff
@@ -1,52 +1,71 @@
 import datetime
+import logging
+
 from celery import shared_task
 from django.db import transaction
-from .models import Task
+
+from _main_.settings import IS_PROD, SLACK_SUPER_ADMINS_WEBHOOK_URL
+from api.services.utils import send_slack_message
 from task_queue.helpers import is_time_to_run
 from .jobs import FUNCTIONS
-from api.services.utils import send_slack_message
-from _main_.settings import SLACK_SUPER_ADMINS_WEBHOOK_URL, IS_PROD, IS_LOCAL, IS_CANARY
+from .models import Task
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+FAILED = "FAILED"
+SUCCEEDED = "SUCCEEDED"
+SKIPPED = "SKIPPED"
+
+def get_task_info(task):
+	if task:
+		return f"task: {task.name}, job: {task.job_name}, task_id: {task.id}"
+	
+def log_status(task, message):
+	task_name = get_task_info(task)
 
-ENV = "API Local" if IS_LOCAL else "API CANARY" if IS_CANARY else "API PROD" if IS_PROD else "API DEV"
+	if task.status == FAILED:
```

Comment:
someones linter isn't going to like these tab indentations.   Better to set them right the first time.